kubernetes/doc/edge-deployment-to-k8s-translations.md

```diff
@@ -0,0 +1,215 @@
+# Conversion of Edge Deployment ModuleSpec to Kubernetes Deployments.
+
+Azure IoT Edge on Kuberenetes project will take an existing Azure IoT Edge application that is 
+running on a single device and install that application into a Kubernetes cluster.  The expectation 
+is that these applications will "just work," regarless of where it is installed.  In order to do 
+that, the Edge on K8s project has to transform the edge deployment into Kubernetes objects which 
+will provide the framework for modules to communicate with other modules.
+
+Existing Edge Deployments running on a single device specifies module setting via the [Docker 
+ContainerCreate structure](https://docs.docker.com/engine/api/v1.40/#operation/ContainerCreate). 
+This means Edge on K8s will have to make decisions about how to provide the module framework based 
+on these settings. This document will describe which values are used from the module specification 
+and how they are transformed into Kubernetes objects.
+
+The application will create Deployments, Services, ImagePullSecrets, PersistentVolumeClaims, and 
+ServiceAccounts to establish this framework.
+Each will be described in detail in subsequent sections.
+
+## Deployment
+
+Each Edge Module will create one Deployment. 
```

Comment:
Please also clarify the namespace part. Each module belongs to one namespace. How is namespace defined/created in K8s.